test/ir_Hitachi_test.cpp

```diff
@@ -1082,9 +1103,8 @@ TEST(TestIRHitachiAcClass424, toCommon) {
   ASSERT_EQ(20, ac.toCommon().degrees);
   ASSERT_EQ(stdAc::opmode_t::kCool, ac.toCommon().mode);
   ASSERT_EQ(stdAc::fanspeed_t::kMax, ac.toCommon().fanspeed);
-  // TODO(jamsinclair): Add support.
-  ASSERT_EQ(stdAc::swingv_t::kOff, ac.toCommon().swingv);
   // Unsupported.
+  ASSERT_EQ(stdAc::swingv_t::kOff, ac.toCommon().swingv);
```

Comment:
Move back above the `// Unsupported` line. We will support it. By hook or by crook! ;-)